### ISSUE TITLE
`defiads-cli`

### ISSUE THREAD

**Issue by ZmnSCPxj** [defiads/defiads#23]
It would be nice to have a simple tool, similar to `bitcoin-cli` or `lightning-cli`, to pass in commands to the `defiads` RPC interface, without having to use cURL or pass in JSON through the shell.

**Comment by philbertw4** [comment]
Yes, I think this is something that we should have sooner than later.

**Comment by philbertw4** [comment]
I'm going to start working on this. It may be ugly and may take me a while due to my n00b rust sklls. If somebody else is already working on this (which would be great!), hopefully they will see issue and chime in.

As for now, my plan is to take a look at a few of the other rust projects in hopes of finding one with a decent `-cli` tool that I can use as a template.

**Comment by philbertw4** [comment]
Just wanted to provide a quick update to this. Sadly, I have not been able to dedicate the time to this that I had hoped, so have not made progress on any sort of defiads-cli yet.
